bindings/go/unicorn/unicorn_test.go

```diff
@@ -24,3 +24,16 @@ func TestMemUnmap(t *testing.T) {
 		t.Fatal(fmt.Errorf("Expected ERR_WRITE_UNMAPPED, got: %v", err))
 	}
 }
+
+func TestDoubleClose(t *testing.T) {
+	mu, err := NewUnicorn(ARCH_X86, MODE_32)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := mu.Close(); err != nil {
+		t.Fatal(err)
+	}
+	if err := mu.Close(); err != nil {
```

Comment:
No, I'm testing to make sure calling Close() twice does NOT error, because I implicitly call it in a GC hook and a user might also call it.